init.fish

```diff
@@ -33,9 +33,9 @@ function fish_user_key_bindings
     and read -l theme < $OMF_CONFIG/theme
     or set -l theme default
   # Prepare packages key bindings paths
-  set -l key_bindings $OMF_CONFIG/key_binding?.fish \
+  set -l key_bindings $OMF_CONFIG/key_binding*.fish \
                       {$OMF_CONFIG,$OMF_PATH}/pkg/*/key_bindings.fish \
-                      {$OMF_CONFIG,$OMF_PATH}/themes/$theme/key_binding?.fish
+                      {$OMF_CONFIG,$OMF_PATH}/themes/$theme/key_binding*.fish
```

Comment:
Switching from `?` to `*` in these leads to a much greater chance of false positives (maybe someone has a `key_bindings_old.fish` file or something) so I don't love it. Maybe moving the `*` into the path somewhere is a better approach?

```
$OMF_CONFIG*/key_bindings.fish \
  {$OMF_CONFIG,$OMF_PATH}/pkg/*/key_bindings.fish \
  {$OMF_CONFIG,$OMF_PATH}/themes*/$theme/key_bindings.fish
```

Reply:
I didn't think of that, good catch! I agree that's a better approach and the result is the same. Thanks!